dispatches/models/nuclear_case/Hydrogen_Turbine/h2_reaction.py

```diff
@@ -0,0 +1,151 @@
+##############################################################################
```

Comment:
Minor comment: There is a generic reaction framework similar to the generic properties framework that allows for easy definition of reactions.